fly/commands/clear_resource_cache.go

```diff
@@ -0,0 +1,65 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/vito/go-interact/interact"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/fly/commands/internal/flaghelpers"
+	"github.com/concourse/concourse/fly/rc"
+)
+
+type ClearResourceCacheCommand struct {
+	Resource 	 flaghelpers.ResourceFlag `short:"r" long:"resource" required:"true" value-name:"PIPELINE/RESOURCE" description:"Name of a resource to clear cache"`
+	Version      *atc.Version             `short:"v" long:"version"                  value-name:"VERSION"           description:"Version of the resource to check from, e.g. digest:sha256@..., in case a version is not specified the command will delete all the resource caches for that resource"`
```

Comment:
There seems to be a little bit of formatting weirdness, I think you can actually fix it by running `go fmt fly/commands/clear_resource_cache.go`!